packages/explorer/src/app/(explorer)/hooks/useIndexerApiUrl.ts

```diff
@@ -0,0 +1,19 @@
+import { anvil, garnet, redstone } from "viem/chains";
+import { useChain } from "./useChain";
+
+const dozerUrl = {
+  [garnet.id]: "https://indexer.mud.garnetchain.com/q",
+  [redstone.id]: "https://indexer.mud.redstonechain.com/q",
```

Comment:
these are on the chain config exported from MUD
https://github.com/latticexyz/mud/blob/6c056de6090a6f4a9633b96513ca1738dc0993c1/packages/common/src/chains/redstone.ts#L7
https://github.com/latticexyz/mud/blob/6c056de6090a6f4a9633b96513ca1738dc0993c1/packages/common/src/chains/garnet.ts#L7

Reply:
perfect, updated